### ISSUE TITLE
Getting error using load batch image node

### ISSUE THREAD

**Issue by cochese9000** [WASasquatch/was-node-suite-comfyui#25]
hi, I'm getting this error when trying batch process a folder of images.

I'm pretty sure the path is right, if I change it, I get a 'folder not found' error.  So it looks like it failing on the image reading or something.

```
Traceback (most recent call last):
  File "P:\ai\ComfyUI\execution.py", line 182, in execute
    executed += recursive_execute(self.server, prompt, self.outputs, x, extra_data)
  File "P:\ai\ComfyUI\execution.py", line 58, in recursive_execute
    executed += recursive_execute(server, prompt, outputs, input_unique_id, extra_data)
  File "P:\ai\ComfyUI\execution.py", line 58, in recursive_execute
    executed += recursive_execute(server, prompt, outputs, input_unique_id, extra_data)
  File "P:\ai\ComfyUI\execution.py", line 58, in recursive_execute
    executed += recursive_execute(server, prompt, outputs, input_unique_id, extra_data)
  File "P:\ai\ComfyUI\execution.py", line 67, in recursive_execute
    outputs[unique_id] = getattr(obj, obj.FUNCTION)(**input_data_all)
  File "P:\ai\ComfyUI\nodes.py", line 129, in encode
    x = (pixels.shape[1] // 64) * 64
AttributeError: 'NoneType' object has no attribute 'shape'
```

Is there a working example of how to use this?  Maybe I'm doing it wrong..

**Comment by WASasquatch** [comment]
The error isn't coming from Load Batch Images but some node in ComfyUI base (line 129). I'll take a look at what that is in ComfyUI when I get at the computer. What are you doing with the images after they load? What other nodes are you using?

**Comment by WASasquatch** [comment]
I can't reproduce this error.

The error comes from the `VAEEncode` node, not WAS Node Suite. It's being fed something that isn't a proper image, it seems like. It should only load image files from the select path you input. But perhaps some of the images are corrupted or damaged? 

PS here is a workflow you can try: https://github.com/WASasquatch/was-node-suite-comfyui/tree/main/workflows#load-batch-imges---incremental-images

**Comment by cochese9000** [comment]
Hi WAS, sorry for delay, I'll get it running again and drop you an exmaple workflow.  BRB.

**Comment by cochese9000** [comment]
Ah,I tried your exmaple file, which was functionally identical and it had the same error, so thanks for that.  I removed any custom nodes and web extensions except yours and it seems to be due to the 'Slarper.IncrementalWidget' extnesion.  Which I only tested briefly, so I've removed it.  This, weirdly, seemed to work once, but then not matter when I did, everything failed with the same error.  

I realised by trial and error, that I needed to recreate some of the nodes to get it working (not sure how many as I did a lot of pruning and re-cloning of comfy and your repo).  I think somehow the workflow.json structure contained erroneous data caused by that plugin or maybe the 'colors.js' plugin I use.  I've since added that and it works fine.

It's all working well now, thanks for your help.
